Documentation/json-schema/distro-support-schema.json

```diff
@@ -2,51 +2,127 @@
-        "properties":
-        {
+        "additionalProperties": false,
+        "properties": {
             "installCommand": {
                 "type": "array",
-                "typeDefinition" : {
-                    "runUnderSudo":
-                    {
-                        "type": "boolean",
-                        "description": "true if this command should be executed under sudo on linux"
-                    },
-                    "commandRoot":
-                    {
-                        "type": "string",
-                        "description": "The first word in the command, e.g. if apt-get install foo, it would be the string apt."
-                    },
-                    "commandParts":
-                    {
-                        "type": "array",
-                        "description": "The remaining parts of the command in an array, for apt-get install {0} it would be [install, {0}] as strings"
-                    },
-                    "description": "An array of these command objects needed to run in-order"
+                "items": {
+                    "$ref": "#/$defs/executable_command"
                 },
                 "description": "The command(s) needed to install dotnet for this distro. Use {0} for the package to install"
             },
             "uninstallCommand": {
                 "type": "array",
+                "items": {
+                    "$ref": "#/$defs/executable_command"
+                },
                 "description": "The command(s) needed to uninstall dotnet. Use {0} for the package"
             },
             "updateCommand": {
                 "type": "array",
+                "items": {
+                    "$ref": "#/$defs/executable_command"
+                },
                 "description": "The command(s) needed to update dotnet. Use {0} for the package"
             },
             "searchCommand": {
                 "type": "array",
+                "items": {
+                    "$ref": "#/$defs/executable_command"
+                },
                 "description": "The command(s) needed to see if a package is available. Use {0} for the package"
             },
             "isInstalledCommand": {
                 "type": "array",
+                "items": {
+                    "$ref": "#/$defs/executable_command"
+                },
                 "description": "The command(s) needed that will return true if dotnet is installed on the machine"
             },
             "packageLookupCommand": {
                 "type": "array",
+                "items": {
+                    "$ref": "#/$defs/executable_command"
+                },
+                "description": "The command(s) needed that will return true if a given package is installed on the machine"
+            },
+            "readSymLinkCommand": {
+                "type": "array",
+                "items": {
+                    "$ref": "#/$defs/executable_command"
+                },

```

Comment:
This is a new kind of command that was in the actual file but not reflected in the schema.

Reply:
Somehow I forgot to add this one. Good catch!